pkg/ui/src/views/reports/containers/chartCatalog/index.tsx

```diff
@@ -0,0 +1,492 @@
+            isLoaded: true,
+            catalog: result.catalog,
+          });
+        },
+        error => {
+          this.setState({
+            isLoaded: true,
+            error,
+          });
+        },
+      );
+    window.scrollTo(0, 0);
+  }
+
+  render() {
+    const { error, isLoaded, catalog, storeMetrics } = this.state;
+    if (error) {
+      return <div>Error: {error.message}</div>;
+    } else if (!isLoaded) {
+      return <div>Loading...</div>;
+    } else {
+      const cat: CatalogAndStoreMetrics = {
+        catalog: catalog,
+        storeMetrics: storeMetrics,
+      };
+      return (
+        <section className="section">
+          <h1>Chart Catalog</h1>
+          <p className="catalog-description">
+            Understand your cluster's behavior with this curated set of charts.
+            <br />
+            <br />
+            <strong className="catalog-link">Bold</strong> links return all •{" "}
+            <span className="catalog-link">Charts</span> under their{" "}
+            <span className="overhang">overhang.</span>
+          </p>
+          <div className="catalog">
+            {_.map(catalog, (sec, key) => (
+              <RenderSection section={sec} key={key} cat={cat} />
+            ))}
+          </div>
+        </section>
+      );
+    }
+  }
+}
+
+function mapStateToProps(state: AdminUIState) {
+  return {
+    nodesSummary: nodesSummarySelector(state),
+  };
+}
+
+const mapDispatchToProps = {
+  refreshNodes,
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps,
+)(withRouter(ChartCatalog));
+
+function RenderSection(props: {
+  section: cockroach.ts.catalog.ChartSection;
+  key: number;
+  cat: CatalogAndStoreMetrics;
+}) {
+  function createMarkup() {
+    return { __html: props.section.description };
+  }
+
+  return (
+    <div className="catalog-section">
+      <table className="catalog-table">
+        <tr>
+          <td className="catalog-table__section-header">
+            <div className="sticky">
+              <h2 className="section-title">{props.section.title}</h2>
+            </div>
+            <p
+              className="section-description"
+              dangerouslySetInnerHTML={createMarkup()}
+            />
+          </td>
+          <td>
+            <div className="catalog-table__chart-links">
+              {_.map(props.section.subsections, (sec, key) => (
+                <RenderSectionTable
+                  section={sec as cockroach.ts.catalog.ChartSection}
+                  key={key}
+                  cat={props.cat}
+                />
+              ))}
+            </div>
+          </td>
+        </tr>
+      </table>
+    </div>
+  );
+}
+
+function RenderSectionTable(props: {
+  section: cockroach.ts.catalog.ChartSection;
+  key: number;
+  cat: CatalogAndStoreMetrics;
+}) {
+  function handleClick(e: React.MouseEvent<HTMLAnchorElement>) {
+    e.preventDefault();
+    window.location.hash = getCustomChartURL(
```

Comment:
It seems like you are using `withRouter` on the `ChartCatalog` to give it react router props, but the doing the hash changes with `window.location`. Why not using react router's [`location`](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/location.md) instead?